src/res/translation/translation_fr_FR.ts

```diff
@@ -663,7 +663,7 @@
     </message>
     <message>
         <location filename="../../clientdlg.cpp" line="172"/>
-        <source>The sound card buffer delay (buffer size) is set to a too small value.</source>
+        <source>The sound card buffer delay (buffer size) is set to too small a value.</source>
```

Comment:
You're 100% sure here? It seems very weird to me.

Reply:
Yes.

Reply:
I just realised I also have to make those changes to quite a few files... ugh!

Reply:
> Yes.

You're right.

Reply:
That would also be correct but I don't understand your problem with how I rephrased it. I don't think a native speaker would find anything strange about it.

Reply:
BTW I'm going to close this pull request and make a new one with all the relevant files corrected.

Reply:
@ignotus666 : because it's sort of "_high level language_" and probably _"the sound card's buffer is too small - increasing it's size would fix the problem_" is the way usually people will speak about it. For a translator, it's usually easier to read from a "_common language_" text rather than a "_high level language_" one.